src/js/badge-card-generator.js

```
@@ -0,0 +1,630 @@
+      const theme = getCurrentTheme();
+
+      // Create temporary container
+      const container = document.createElement('div');
+      container.style.position = 'fixed';
+      container.style.left = '-9999px';
+      container.style.top = '0';
+      container.innerHTML = createCardHTML(badgeData, theme);
+      document.body.appendChild(container);
+
+      // Generate canvas
+      const canvas = await html2canvas(container.firstElementChild, {
```

ESLint

[error] 'html2canvas' is not defined.

(no-undef)